incubator/prometheus-alerts/values.yaml

```diff
@@ -0,0 +1,10 @@
+# Default values for prometheus-alerts.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+labels:
+  app: prometheus
+  # In common case prometheus name is release name of chart that install prometheus
+  prometheus:
+level:
```

Comment:
I don't like that `level` is generalized across all `prometheus-alerts`. Maybe we can have `defaults`, but there should still be the notion of specifying thresholds per alert.  In this case, I think it should be something like

```
alerts:
  cluster_cpu_low:
     warning: 40
     critical: 20
```

... also I need to think if about the naming. Seems like `cluster_cpu_low` is too vague.